src/WordPress/PluginActions.php

```diff
@@ -27,6 +28,7 @@ public function __construct(DefaultIntegration $defaultIntegration, APIInterface
         $this->wordpressAPI = $defaultIntegration->getIntegrationAPI();
         $this->dataStore = $defaultIntegration->getDataStore();
         $this->logger = $defaultIntegration->getLogger();
+        $this->integration = $defaultIntegration;
```

Comment:
Instead of doing this lets extend the core functionality. 
1. Create `CF\WordPress\Router\WordPressPluginAPIRouter.php`
2. Extend `route(Request $request)` to pass in a `CF\WordPress\API\WordPressClientAPI`
3. Update this constructor to accept `CF\WordPress\API\WordPressClientAPI`
4. `$this->wordPressAPIClient = $wordPressAPIClient